### PR TITLE
Added Sloan grant PIs

### PR DESCRIPTION
**Pull request by JeffCarver**
@cosden Please have a look

### CODE REVIEW

**Review by cosden — APPROVED** [review]

**Comment by cosden** [comment]
This looks good to me. Let's check with @danielskatz before merging since it has his name on it too.

File: pages/committee.md [hunk 1]
```diff
@@ -14,3 +14,8 @@ permalink: /committee/
 1. Christina Maimone, Northwestern University
 1. Micaela Parker, Academic Data Science Alliance
 1. Jordan Perr-Saur, National Renewable Energy Lab
+
+## Alfred P. Sloan Foundation Grant PIs
+1. Jeffrey C. Carver, University of Alabama
+1. Ian Cosden, Princeton University
+1. Daniel S. Katz, University of Illinois Urbana-Champaign
```
Comment (danielskatz) [inline pages/committee.md]:
```suggestion
1. Daniel S. Katz, University of Illinois at Urbana-Champaign
```

**Review by danielskatz — APPROVED** [review]
lgtm

**Comment by danielskatz** [comment]
I'll go ahead and merge this

**Pull request merged**